jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -1525,7 +1525,8 @@ ecma_op_function_try_to_lazy_instantiate_property (ecma_object_t *object_p, /**<
     bytecode_data_p = ecma_op_function_get_compiled_code ((ecma_extended_object_t *) object_p);
 
 #if ENABLED (JERRY_ESNEXT)
-    if (!(bytecode_data_p->status_flags & CBC_CODE_FLAGS_STRICT_MODE))
+    if (!(bytecode_data_p->status_flags & CBC_CODE_FLAGS_STRICT_MODE)
+        && !(CBC_FUNCTION_GET_TYPE (bytecode_data_p->status_flags) == CBC_FUNCTION_ARROW))
     {
       ecma_property_t *value_prop_p;
       /* The property_name_p argument contans the name. */
```

Comment:
If you already touched this code, could you please fix this typo too? contans -> contains